src/main/java/org/icgc_argo/workflowgraphnode/components/Errors.java

```diff
@@ -11,6 +11,19 @@
 @Slf4j
 public class Errors {
 
+  /**
+   * Biconsumer for onErrorContinue handlers which handles transaction based on Exception rules
+   * (rules: https://github.com/icgc-argo/workflow-graph-lib#exceptions)
+   *
+   * <p>Notes:
+   *
+   * <p>- onErrorContinue is only supported with some operators, look for operators marked with
+   * "Error Mode Support" in reactor documentation.
+   *
+   * <p>- onErrorContinue behaves differently with Mono since there is only one and nothing to
+   * continue. See Mono doc for more info:
+   * https://projectreactor.io/docs/core/3.3.2.RELEASE/api/reactor/core/publisher/Mono.html#onErrorContinue-java.lang.Class-java.util.function.BiConsumer-
+   */
```

Comment:
I had an issue with this in the activationFunction error handling because of the way `onErrorContinue` works. I noted some of the behaviour that I found here. We should probably be careful using it since it looks like there are some known odd behaviours in reactor-core:  https://github.com/reactor/reactor-core/issues/2184

Reply:
great research!